tests/e2e/operator/operator_suite_test.go

```diff
@@ -50,14 +45,13 @@ func setup() {
 	GinkgoWriter.Println("************ Running Setup ************")
 
 	GinkgoWriter.Println("Initializing k8s client")
+	var err error
 	cl, err = k8sclient.InitK8sClient()
 	Expect(err).NotTo(HaveOccurred())
 
 	if ocp {
 		GinkgoWriter.Println("Running on OCP cluster")
-		GinkgoWriter.Printf("Absolute Path: %s\n", wd)
 	} else {
 		GinkgoWriter.Println("Running on Kubernetes")
-		GinkgoWriter.Printf("Absolute Path: %s\n", wd)
```

Comment:
@fjglira Do we really need this?

Reply:
Nope, Is not needed. Was there for debugging purpose only